### DISCUSSION TITLE
unable to install virtual env on windows

### DISCUSSION THREAD

**Discussion by Baizkunthaz** [learningequality/kolibri#11643] [Q&A]
Someone please guide me to install virtualenv on Windows for Kolibri as I want to contribute, but I am new to it, so please guide me. I have researched on YouTube and Google, but nothing was found helpful to me, and somehow, I have installed virtual env in a folder. I had also added the virtual env files on it, but after that, the required process, which is further needed for the complete setup, doesn't undergo after, so I guess installing virtual env for Kolibri especially on Windows has some other method, so kindly guide me. THANKS!

**Comment by MisRob** [comment]
Hi @Baizkunthaz I will close this since I see you're already chatting about your setup here https://github.com/learningequality/kolibri/discussions/11648
